Rules/UseConsistentIndentation.cs

```diff
@@ -284,6 +289,35 @@ private static bool PipelineIsFollowedByNewlineOrLineContinuation(Token[] tokens
             return false;
         }
 
+        private static bool PreviousLineEndedWithPipe(Token[] tokens, int tokenIndex, Token token)
+        {
+            if (tokenIndex < 2 || token.Extent.StartLineNumber == 1)
+            {
+                return false;
+            }
+
+            int searchIndex = tokenIndex - 2;
+            int searchLine;
+            do
+            {
+                searchLine = tokens[searchIndex].Extent.StartLineNumber;
```

Comment:
Oh! My `switch` suggestion got lost! Ugh, too much context switching... Anyway, not an issue

Reply:
I thought I addressed your comment here by not initializing it: https://github.com/PowerShell/PSScriptAnalyzer/pull/1497#discussion_r439068047